.github/merge_rules.json

```diff
@@ -42,42 +52,25 @@
    },
    {
       "name": "OSS CI",
-      "patterns": [".github/**", ".circleci/**", ".jenkins/**", "scripts/**", "tools/**"],
-      "approved_by": ["ezyang", "pytorch/pytorch-dev-infra"],
-      "mandatory_checks_name": [
-         "Facebook CLA Check",
-         "Lint",
-         "pull"
-      ]
-   },
-   {
-      "name": "CI Pinned Hashes",
       "patterns": [
-         ".github/ci_commit_pins/vision.txt",
-         ".github/ci_commit_pins/torchdynamo.txt"
+         ".github/**",
+         ".circleci/**",
+         ".jenkins/**",
+         "scripts/**",
+         "tools/**",
+         ".github/ci_commit_pins/*"
       ],
-      "approved_by": ["pytorchbot", "ezyang", "pytorch/pytorch-dev-infra"],
+      "approved_by": ["alband", "dagitses", "pytorchbot", "pytorch/pytorch-dev-infra"],
```

Comment:
This change make "pytorchnbot" as an approved reviewer for everythign in CI, but for security/sanity reasons I would prefer bot to be limited to just pinned hashes

Reply:
done